api/_sync-prisma.js

```diff
@@ -1,7 +1,7 @@
 const { execSync } = require("child_process");
 
 execSync("prisma generate", log);
-execSync("prisma migrate dev", log);
+//execSync("prisma migrate dev", log);
```

Comment:
Because sync-prisma is only useful is we have a DB that requires migrations, we can just delete _sync-prisma and remove the script from package.json. 
Instead maybe add a prisma:generate or something that just does a single `prisma generate`.